int32[] stepper_vel
int32[] servo_torque
